alyssabull/idk
# Refactor/app

## What does this PR do?
- Refactors localStorage into helper functions
- Refactors nav bar into `NavBar.js` component
- Adds final styles and universal button hover state
- Exports dropdown info into a separate file

## Where should the reviewer start?
- `NavBar.js`, `dropdownData.js`, and `App.js`

## How should this be manually tested?
- Refresh the page on the random activity and saved activity page to ensure activities persist on page refresh
- Observe button styles and hover states
- Ensure nav bar and dropdown components still function after being extracted into different files
